.github/workflows/install.yml

```diff
@@ -26,7 +26,7 @@ jobs:
   install:
     runs-on: ubuntu-latest
     env:
-      SSH: ssh -o StrictHostKeyChecking=no -i ~/.ssh/sandbox.key ubuntu@next-release.openedx.org
+      SSH: ssh -o StrictHostKeyChecking=no -o TCPKeepAlive=yes -o ServerAliveInterval=120 -i ~/.ssh/sandbox.key ubuntu@next-release.openedx.org
```

Comment:
curious: why choose serveralive over clientalive? Or why not both?

Reply:
I think clientalive is meant for the server-side configuration.  We can add it there, too!

Reply:
Oh, it's alright. I missed the client and server part in docs 🤦🏽 . ServerAlive is on client and vice versa.